core/templates/dev/head/components/answer_group_editor_directive.html

```diff
@@ -92,4 +92,11 @@
                   outcome="outcome"
                   suppress-warnings="suppressWarnings()">
   </outcome-editor>
+
+  <br ng-if="!defaultAnswerGroup">
```

Comment:
See line 46 for a comment on how to determine whether this is a default answer group or not. Define `$scope.answerGroupIsDefault = ($scope.rules !== null);` in the directive and use that throughout; don't pass an additional attribute in.

Reply:
Done!